packages/terra-functional-testing/tests/jest/commands/utils/ScreenshotRequestor.test.js

```diff
@@ -20,6 +26,7 @@ describe('ScreenshotRequestor', () => {
   const oldCwd = process.cwd;
   beforeEach(() => {
     mockInfo.mockClear();
+    jest.resetModules();
```

Comment:
+1 on resetting before each test

Reply:
So `beforeEach` should run on before each test inside the describe block. Since its on the outermost describe block it should run before each test.